core/src/toga/platform.py

```diff
@@ -110,3 +110,11 @@ def get_platform_factory() -> ModuleType:
             backend = matching_backends[0]
         factory = importlib.import_module(f"{backend.value}.factory")
     return factory
+
+
+def __getattr__(name):
+    if name == "backend":
+        globals()["backend"] = get_platform_factory().__package__
+        return globals()["backend"]
```

Comment:
```suggestion
        global backend
        backend = get_platform_factory().__package__
        return backend
```
I know the behavior of the namespace dicts is fully defined now, but I'd still prefer using the dedicated syntax the language gives us for this. `__getattr__` will only be called if the name doesn't already exist.